src/components/region-common.js

```diff
@@ -1,74 +1,77 @@
+      ['EUROPE-WEST1', () => ({ flag: '🇧🇪', regionDescription: `${regionDescription} (Belgium)`, computeZone: 'EUROPE-WEST1-B', computeRegion: 'EUROPE-WEST1' })],
+      ['EUROPE-WEST2', () => ({ flag: '🇬🇧', regionDescription: `${regionDescription} (London)`, computeZone: 'EUROPE-WEST2-A', computeRegion: 'EUROPE-WEST2' })],
+      ['EUROPE-WEST3', () => ({ flag: '🇩🇪', regionDescription: `${regionDescription} (Frankfurt)`, computeZone: 'EUROPE-WEST3-A', computeRegion: 'EUROPE-WEST3' })],
+      ['EUROPE-WEST4', () => ({ flag: '🇳🇱', regionDescription: `${regionDescription} (Netherlands)`, computeZone: 'EUROPE-WEST4-A', computeRegion: 'EUROPE-WEST4' })],
+      ['EUROPE-WEST6', () => ({ flag: '🇨🇭', regionDescription: `${regionDescription} (Zurich)`, computeZone: 'EUROPE-WEST6-A', computeRegion: 'EUROPE-WEST6' })],
+      ['NORTHAMERICA-NORTHEAST1', () => ({ flag: '🇨🇦', regionDescription: `${regionDescription} (Montreal)`, computeZone: 'NORTHAMERICA-NORTHEAST1-A', computeRegion: 'NORTHAMERICA-NORTHEAST1' })],
+      ['NORTHAMERICA-NORTHEAST2', () => ({ flag: '🇨🇦', regionDescription: `${regionDescription} (Toronto)`, computeZone: 'NORTHAMERICA-NORTHEAST2-A', computeRegion: 'NORTHAMERICA-NORTHEAST2' })],
+      ['SOUTHAMERICA-EAST1', () => ({ flag: '🇧🇷', regionDescription: `${regionDescription} (Sao Paulo)`, computeZone: 'SOUTHAMERICA-EAST1-A', computeRegion: 'SOUTHAMERICA-EAST1' })],
+      ['US-CENTRAL1', () => ({ flag: '🇺🇸', regionDescription: `${regionDescription} (Iowa)`, computeZone: 'US-CENTRAL1-A', computeRegion: 'US-CENTRAL1' })],
+      ['US-EAST1', () => ({ flag: '🇺🇸', regionDescription: `${regionDescription} (South Carolina)`, computeZone: 'US-EAST1-B', computeRegion: 'US-EAST1' })],
+      ['US-EAST4', () => ({ flag: '🇺🇸', regionDescription: `${regionDescription} (Northern Virginia)`, computeZone: 'US-EAST4-A', computeRegion: 'US-EAST4' })],
+      ['US-WEST1', () => ({ flag: '🇺🇸', regionDescription: `${regionDescription} (Oregon)`, computeZone: 'US-WEST1-A', computeRegion: 'US-WEST1' })],
+      ['US-WEST2', () => ({ flag: '🇺🇸', regionDescription: `${regionDescription} (Los Angeles)`, computeZone: 'US-WEST2-A', computeRegion: 'US-WEST2' })],
+      ['US-WEST3', () => ({ flag: '🇺🇸', regionDescription: `${regionDescription} (Salt Lake City)`, computeZone: 'US-WEST3-A', computeRegion: 'US-WEST3' })],
+      ['US-WEST4', () => ({ flag: '🇺🇸', regionDescription: `${regionDescription} (Las Vegas)`, computeZone: 'US-WEST4-A', computeRegion: 'US-WEST4' })],
+      [Utils.DEFAULT, () => ({ flag: unknownRegionFlag, regionDescription, computeZone: 'UNKNOWN', computeRegion: 'UNKNOWN' })]
+    )],
+    [Utils.DEFAULT, () => ({ flag: unknownRegionFlag, regionDescription, computeZone: 'UNKNOWN', computeRegion: 'UNKNOWN' })]
+  )
 }
+
+export const allRegions = [
+  // In this list, us-east*, us-west*, northamerica-northeast2 and asia-northeast2 have purposefully been removed.
```

Comment:
Not a hard objection, just trying to understand. Is this restriction geared more toward workspace buckets? Should we allow the full list (us-east, etc) for cloud environments given we allow overriding from the "default" region anyway?

From my initial testing, I tried to create a cloud environment and was confused why there were so many `asia-` options and only 1 `us-` option. I imagine some users might be confused too.

Reply:
Its a good question, and one that we'll have answered in a Terra article (currently under review). Pasting the relevant part here:

> For many organizations and individual labs, the best long-term choice is to select a single region for bucket location. In the US, Terra makes available the us-central1 region(*).
> 
> (*)For data in the US, the Terra interface supports creating workspace buckets in US multi-region or us-central1. Terra's default region for Workflow and Cloud Environment VMs has historically been us-central1. The exclusion of us-east and us-west regions is intended to help the community avoid unintentionally creating data silos in different regions since cross-analysis of data in different regions is inherently more expensive due to network egress charges. If you have a specific need to create workspaces with buckets in these other US regions, please contact Terra Support.

TL;DR: We want to avoid silos of different regions within a country. 

I'm debating how important it is to put this information directly in the UI vs just linking to the proper articles.